docs/WritingTests.asciidoc

```diff
@@ -428,24 +428,22 @@ Children B, C, D can run and finish anytime, but A must run until all B, C, D fi
 
 === Test synchronization and locking API
 
-OpenQA provides locking server through lock API. To use lock API import +lockapi+ package (_use lockapi;_) in your test file.
-Lock API provides functions: +mutex_create+, +mutex_lock+, +mutex_unlock+, +mutex_wait+. Each of these functions take at least one parameter: name of the lock. Note that lock name can't contain "-" character.
-Locks are associated with caller's job - locks can't be unlocked by different job then the one who locked the lock.
+OpenQA provides a locking server. To use it in your test files import +lockapi+ package (_use lockapi;_). It provides the following functions: +mutex_create+, +mutex_lock+, +mutex_unlock+, +mutex_wait+
 
-+mutex_lock+ tries to lock the mutex lock for caller's job. If lock is unavailable or locked by someone else, +mutex_lock+ call blocks.
+Each of these functions takes the name of the lock as first parameter. The name must not contain the "-" character.
+Locks are associated with the caller's job and can not be unlocked by a different job.
```

Comment:
that is wrong. this is actually how it works . the job A which needs to wait for some event provided by job B is calling `mutex_wait` and waits until job A will do what expected and call `mutex_create` afterwards. `mutex_wait` is actually wrapper over lock/unlock function calls

Reply:
Ok. Then I'll remove the sentence "If lock is unavailable or locked by someone else, +mutex_lock+ call blocks." completely rather than rephrasing it. The same is explained in the next paragraph (specific to `mutex_lock`) anyways.